src/main/java/nextstep/subway/favorite/dto/FavoriteResponse.java

```diff
@@ -1,35 +1,35 @@
 package nextstep.subway.favorite.dto;
 
 import nextstep.subway.favorite.domain.Favorite;
-import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.dto.StationResponse;
 
 public class FavoriteResponse {
   private Long id;
-  private Station source;
-  private Station target;
+  private StationResponse source;
+  private StationResponse target;
 
   public FavoriteResponse() {
   }
 
-  public FavoriteResponse(Long id, Station source, Station target) {
+  public FavoriteResponse(Long id, StationResponse source, StationResponse target) {
```

Comment:
```suggestion
  private FavoriteResponse(Long id, StationResponse source, StationResponse target) {
```

아래에 정적 팩토리 메서드가 있으니 다음 미션 진행하실때도 습관적으로 챙겨주시면 좋을 것 같아요 :)

Reply:
아하 넵넵 습관 들이기가 어렵네요.. ㅎㅎㅎ 감사합니다!! 안까먹고 해보겠습니다!